instrumentation/rails/lib/opentelemetry/instrumentation/rails/instrumentation.rb

```diff
@@ -14,19 +14,12 @@ module Rails
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         MINIMUM_VERSION = Gem::Version.new('5.2.0')
 
-        install do |_config|
-          OpenTelemetry::Instrumentation::ActionPack::Instrumentation.instance.install({})
-          OpenTelemetry::Instrumentation::ActionView::Instrumentation.instance.install({})
-          OpenTelemetry::Instrumentation::ActiveRecord::Instrumentation.instance.install({})
-        end
```

Comment:
This change is required as with it being present you're unable to disable the via config being passed in.  It's more consistent with the `all` instrumentation gem than any of the other ones so I think that's ok.  Happy to bikeshed on this.

Reply:
If this is no longer auto-installing sub-gems, should it also remove requires?

https://github.com/open-telemetry/opentelemetry-ruby/blob/fix-remove-explicit-installs/instrumentation/rails/lib/opentelemetry/instrumentation/rails.rb#L18

Reply:
I think we want it to still require the instrumentation sub components to make sure that the instrumentation hooks get called.  We do something similar in [`instrumentation-all`](https://github.com/open-telemetry/opentelemetry-ruby/blob/9e86944a0347a63b393246cf94d585e04e17b539/instrumentation/all/lib/opentelemetry/instrumentation/all.rb#L7-L35) which is where I pulled my inspiration from.